exercises/B-hello-world/exercise.js

```diff
@@ -1 +1 @@
-console.log("Hello world");
+console.log("Hello world-2");
```

Comment:
You may also try to print several things within the same console log, for example: 

console.log(
  "Hello World. I just started learning JavaScript!",
  "Second item",
  99
);